src/gui/menu_species.cpp

```diff
@@ -363,3 +446,24 @@ void DissolveWindow::on_SpeciesScaleChargesAction_triggered(bool checked)
 
     fullUpdate();
 }
+
+void DissolveWindow::on_SpeciesReduceChargesSigFigsAction_triggered(bool checked)
+{
+    // Get the current Species (if a SpeciesTab is selected)
+    auto species = ui_.MainTabs->currentSpecies();
+    if (!species)
+        return;
+
+    auto ok = false;
+    auto significantFigures =
+        QInputDialog::getInt(this, "Reduce Signficant Figures in Charges",
+                             "Enter the number of significant figures to use for all atoms", 1, 1, 100, 1, &ok);
+    if (!ok)
+        return;
+
+    for (auto &atom : species->atoms())
+    {
+        double rounded = std::round(atom.charge() * std::pow(10, significantFigures)) / std::pow(10, significantFigures);
+        atom.setCharge(rounded);
+    }
+}
```

Comment:
The temporary variable is a bit redundant:
```suggestion
    for (auto &atom : species->atoms())
        atom.setCharge(std::round(atom.charge() * std::pow(10, significantFigures)) / std::pow(10, significantFigures));
```